wrapper/pre_copy.py

```diff
@@ -137,6 +137,8 @@ def __init__(self, data):
                 self.insecure = False
             else:
                 raise ValueError('Invalid value for "no_verify"')
+        elif data['transport_method'] == 'ssh':
+            self.insecure = True
```

Comment:
Is the reason for this that connecting with `esx://` won't have any valid certificate? Or is it possible to have a setup where certificate validation would work securely?